lhotse/cut.py

```diff
@@ -505,17 +491,27 @@ def load_features(self, root_dir: Optional[Pathlike] = None) -> np.ndarray:
             )
         return mixer.mixed_feats
 
-    def load_audio(self,
-                   recording_set: Optional[RecordingSet] = None,
-                   root_dir: Optional[Pathlike] = None,
-    ) -> List:
-        """Loads the audios of the source cuts and put them into a list."""
+    def load_unmixed_audio(self, root_dir: Optional[Pathlike] = None) -> List:
```

Comment:
For consistency, it could return a numpy array with shape (n_channels, n_samples)

Reply:
The lengths of the audio are different, so we have to pad them to put them into a matrix. Returning as List avoids padding. So I prefer the current implement ...

Reply:
To make sense of that, wouldn't you need to know the offsets?   That would be complex for the user, no?
I kind of feel like padding them to the same length would be the cleanest/most useful solution.

Reply:
.. and please clarify that this is before applying the SNRs?

Reply:
Mm.. I think it might be useful to have an option to have the SNRs applied already, as they would be
in 'load_audio' except keeping things in separate channels?  Is this complex to implement?

Reply:
I think there is another solution: just remove this `load_unmixed_audio()`, only retain the `load_unmixed_audio()` which returns the mixed audio in an `np.ndarray`, just like what `load_features()` does. This way can avoid users being confused. Not sure it this solution appropriate ...